src/NHibernate/Proxy/NHibernateProxyBuilder.cs

```diff
@@ -24,7 +23,6 @@ internal class NHibernateProxyBuilder
 		private static readonly PropertyInfo LazyInitializerIdentifierProperty = LazyInitializerType.GetProperty(nameof(ILazyInitializer.Identifier));
 		private static readonly MethodInfo LazyInitializerInitializeMethod = LazyInitializerType.GetMethod(nameof(ILazyInitializer.Initialize));
 		private static readonly MethodInfo LazyInitializerGetImplementationMethod = LazyInitializerType.GetMethod(nameof(ILazyInitializer.GetImplementation), System.Type.EmptyTypes);
-		private static readonly IProxyAssemblyBuilder ProxyAssemblyBuilder = new DefaultProxyAssemblyBuilder();
```

Comment:
I have not seen any interest in keeping the `NHibernate.Proxy.DynamicProxy.IProxyAssemblyBuilder` pattern now that it was used only by non-injectable private members. So I have moved its features to `ProxyBuilderHelper`.